bots/jabber_mirror_backend.py

```diff
@@ -90,15 +96,18 @@ def __init__(self, jid, password, rooms):
         self.register_plugin('xep_0199') # XMPP Ping
 
     def set_zulip_client(self, client):
+        # type: (zulip.Client) -> None
         self.zulip = client
 
     def session_start(self, event):
+        # type: (Any) -> None
```

Comment:
The event argument in that function is redundant since it is not used anywhere in the function's body, so I assume it can be safely set to Any.